docs/sections/getting_started/setup.md

```diff
@@ -15,8 +15,65 @@ notebookmetadatakey2: val2
 
 (fundamentals-setup)=
 
-# Setup
+# Setup on AiiDAlab
 
-(fundamentals-setup-profile)=
+It is assumed that most users will find it easiest to follow the tutorial on the dedicated AiiDAlab instance accessible at [https://aiida-intro-tutorial-2021.aiidalab.net](https://aiida-intro-tutorial-2021.aiidalab.net).
+This instance of AiiDAlab is pre-configured with all the software needed for the tutorial and is the fastest way to get started for anyone who does not already have AiiDA installed and configured on their own machine.
 
-## Profile
+This is how you access the server:
+
+  1. If you do not have a GitHub account yet, create one at [https://github.com/login](https://github.com/login).
+  2. Open [https://aiida-intro-tutorial-2021.aiidalab.net](https://aiida-intro-tutorial-2021.aiidalab.net) in your browser (we tested Firefox and Chrome).
+  3. Click on "Sign in with GitHub".
+  4. At this point you might be asked to log into your GitHub account unless you are already logged in.
+     If you do not have an account yet, create one now.
+  5. When asked to authorize the "AiiDA Intro Tutorial 2021" app, click on "Authorize aiidalab".
+  6. The server is then prepared for your account, and you will be redirected to the JupyterLab interface after 1-2 minutes.
+
+Subsequent logins should take you directly to the server as long as you have cookies enabled within your browser.
+
+```{figure} include/images/first-login.gif
+:align: center
+
+Demonstration of the (first) login procedure.
+
+```
+
+```{important} The tutorial server will be kept online for a few more days after the tutorial has eneded.
```

Comment:
```suggestion
```{important} The tutorial server will be shut down on **Friday July 9 at 23:59 CEST**.
```

Reply:
Last time we kept it online for a few days. You sure you want to shut it down so soon?

Reply:
I'm not sure, would have to discuss with @giovannipizzi. Note that

1. Friday is a hackaton, where they should be working on their own machine after the installation session on Thursday
2. We can always extend the deadline